src/components/Image/ImageBanner/ImageBanner.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import Image from 'components/Image/Image.js';
+
+function ImageBanner({ image_banner }) {
+   return (
+    <div className="image-banner">
+      {image_banner.map((el, i) =>  
+       <Image key={i} className="image-banner__image" img={el.url} alt={el.alt}></Image>      
+      )}
+    </div>
+   );
+} 
```

Comment:
to be consistent with the rest of the project we should use arrow functions, also our argument should be a bit more descriptive, like images, and image banner takes images instead of an image banner 😄 

```suggestion
const ImageBanner = ({ images }) => {
   return (
    <div className="image-banner">
      {images.map((el, i) =>  
       <Image key={i} className="image-banner__image" img={el.url} alt={el.alt}></Image>      
      )}
    </div>
   );
} 
```